FAQ.md

```diff
@@ -1,202 +1,122 @@
->
-> The Odin Project isn't like college or university, TOP is self paced and allows you to get a really solid grasp of concepts before moving on. In school, you're forced to keep up or you will fail. The difference here is that coming into TOP, you're not expected to have much knowledge, there's no prerequisites. We've had people be successful coming through here that only knew how to check their email with a computer, we've also seen success from computer science degree holders. Treating TOP like a static timeline is ignorant simply because you don't know what you don't know yet, and that's OK! There's no due-dates on things in The Odin Project so you can spend the time to do it right and discuss the topics.
+There is no data on this information, between 3 months and 3 years seems to be the range. More info [here](https://discord.com/channels/505093832157691914/505093832157691916/765633002393829389).
 
 **Can I contribute?**
 
-Of course, check out our github and follow the contributing guide: https://github.com/TheOdinProject
+Of course, check out [our Github](https://github.com/TheOdinProject) and follow the contributing guide.
 
 **What is Thinkful and their relationship to TOP?**
-Thinkful pays the bills around here, it was part of the deal when Erik sold Viking Code School to Thinkful. They don't ask us for anything in return except a small ad on the site. They do not use our curriculum, nor do they contribute to ours. TOP is fully run by the Core and Maintainer teams, as well as other contributors on a volunteer basis. 
 
-**Are you affiliated with "\_\_\_\_\_ bootcamp"?**
+Thinkful pays for the domain and servers, it was part of the deal when Erik sold Viking Code School to Thinkful. They don't ask us for anything in return except a small ad on the site. They do not use our curriculum, nor do they contribute to ours. TOP is fully run by the Core and Maintainer teams, as well as other contributors on a volunteer basis.
 
-Very most likely not, see this: 
+**Are you affiliated with "_ bootcamp"?** 
 
-> Microverse is using our content without permission
-> 
-> they are making money from our stuff without permission
-> 
-> We've asked them to stop as well. They will not
-> 
-> Microverse has no permission from us to use our hard work for their ill gained funds. They're expensive as well, especially for a program that uses students to teach the next ones.
-> It's scummy, and anti-consumer
-![image](https://user-images.githubusercontent.com/4411701/134785911-d3da6847-6453-4ea5-93b3-9c241580bac4.png)
+Unless it is Thinkful, no. [Read here for more information](https://discord.com/channels/505093832157691914/505093832157691916/781170792095809547).
```

Comment:
I'm not sure how many times MV has been mentioned lately. It seems to be a lot less compared to when I started out. If it doesn't seem to be a problem anymore I'd vote to remove it.

Reply:
Just my 2 pence but I'd keep the MV bit. It might sadly be the case that in the future (or even currently and we don't know about it) does the same thing and someone seeing this bit may start wondering about that MV-like boot camp they're going through.

Reply:
I guess that might be more reason to consider removing it then. I remember it being brought up daily last year. Now I hardly ever see it come up.